packages/devtools/tool/analysis_options_check.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
why not make this a regular test as part of the devtools repo?

Reply:
Devon and I discussed putting it in its own dart script because it seemed similar to the version_check we have. Where are you proposing we put this check?